layouts/research_lines/single.html

```diff
@@ -0,0 +1,111 @@
+{{- define "main" -}}
```

Comment:
This file should be named terms.html since this is a taxonomy. This also prevents you from having to set layout: single everywhere

Reply:
This should be resolved now (before previous meeting)